kyuubi-common/src/main/scala/org/apache/kyuubi/config/KyuubiConf.scala

```diff
@@ -823,6 +824,11 @@ object KyuubiConf {
       "          <li><code>KERBEROS</code></li>" +
       "        </ul>" +
       "      </li>" +
+      "      <li>SASL/OAUTHBEARER" +
+      "        <ul>" +
+      "          <li><code>BEARER</code></li>" +
+      "        </ul>" +
+      "      </li>" +
       "    </ul>" +
```

Comment:
Hi @George314159 thanks for raising this PR.

Some comments, for current design:
we do not need to add new item into kyuubi.authentication, for example, now I set `kyuubi.authentication=KERBEROS,CUSTOM` in server end, for `Bearer` authentication, it belongs to `CUSTOM` authentication.